suggestions/contacts.json

```diff
@@ -1 +1,16 @@
-[]
\ No newline at end of file
+[
+    {
+        "id": "com.simplemobiletools.contacts.pro",
+        "label": "Simple Contacts Pro",
+        "source": "fg",
+        "reason": "Contacts address book for managing recent contacts and vcf contact backups",
+        "repo": "https://github.com/SimpleMobileTools/Simple-Contacts"
+    },
```

Comment:
Does not follow RFC standard for storing contacts which is necessary for third-party syncing to work properly. See https://github.com/SimpleMobileTools/Simple-Contacts/issues/364

Reply:
In order to fix this issue, a fork of this app has been created, until the original one merges these features: [Simple Contacts SE](https://github.com/stephanritscher/Simple-Contacts)

Reply:
> In order to fix this issue, a fork of this app has been created, until the original one merges these features: [Simple Contacts SE](https://github.com/stephanritscher/Simple-Contacts)

Read the issue above. It may never be merged by the project. In any case, we do need a third-party app, and **Simple Contacts SE** seems to be a better choice than _Simple Contacts Pro_ right now.

Reply:
Simple Contacts Pro will be replaced by Simple Contacts SE in the next commits